lib/Hakyll/Web/Feed.hs

```diff
@@ -176,8 +220,32 @@ renderAtom :: FeedConfiguration       -- ^ Feed configuration
 renderAtom = renderAtomWithTemplates atomTemplate atomItemTemplate
 
 
+--------------------------------------------------------------------------------
+-- | Render a JSON feed with a number of items.
+renderJsonFeed :: FeedConfiguration       -- ^ Feed configuration
```

Comment:
Other rendering functions don't have "Feed" in their hame, can you please change this one to match?